adapters/teads/teads.go

```diff
@@ -183,14 +187,16 @@ func getTeadsRendererFromBidExt(ext json.RawMessage) (*teadsBidExt, []error) {
 	return &bidExtTeads, nil
 }
 
-func getMediaTypeForImp(impID string, imps []openrtb2.Imp) openrtb_ext.BidType {
+func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, []error) {
 	for _, imp := range imps {
 		if imp.ID == impID {
 			if imp.Video != nil {
-				return openrtb_ext.BidTypeVideo
+				return openrtb_ext.BidTypeVideo, nil
 			}
-			return openrtb_ext.BidTypeBanner
+			return openrtb_ext.BidTypeBanner, nil
 		}
 	}
-	return openrtb_ext.BidTypeBanner
+	return openrtb_ext.BidTypeBanner, []error{&errortypes.BadInput{
+		Message: "Imp ids were not equals",
+	}}
```

Comment:
could return `openrtb_ext.BidType("")`  instead of `openrtb_ext.BidTypeBanner`
```
openrtb_ext.BidType(""), []error{&errortypes.BadInput{
		Message: "Imp ids were not equals",
}}
```